bpf/aya-bpf/src/maps/xdp/dev_map.rs

```diff
@@ -0,0 +1,63 @@
+use core::{cell::UnsafeCell, mem, ptr::NonNull};
+
+use aya_bpf_bindings::bindings::bpf_devmap_val;
+use aya_bpf_cty::c_void;
+
+use crate::{
+    bindings::{bpf_map_def, bpf_map_type::BPF_MAP_TYPE_DEVMAP},
+    helpers::{bpf_map_lookup_elem, bpf_redirect_map},
+    maps::PinningType,
+};
+
+#[repr(transparent)]
+pub struct DevMap {
```

Comment:
And here.

Reply:
Done in 1fbb1d5f70096eb634816730b69882ff3039877c.